<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite hostname="DESKTOP-V4U71TT" failures="1" tests="7" name="com.tutorialsninja.testcases.Logintest" time="70.090" errors="0" timestamp="2023-07-06T07:57:41 IST" skipped="0">
  <testcase classname="com.tutorialsninja.testcases.Logintest" name="verify_login_with_valid_credencials" time="5.142"/>
  <system-out/>
  <testcase classname="com.tutorialsninja.testcases.Logintest" name="verifyloggingintoapplicationusingkeyboardkeys" time="3.196"/>
  <system-out/>
  <testcase classname="com.tutorialsninja.testcases.Logintest" name="verify_forgetpasswordlink_in_loginpage" time="21.229"/>
  <system-out/>
  <testcase classname="com.tutorialsninja.testcases.Logintest" name="verify_Loggingino_the_application_and_browsingback_using_browserback_button" time="7.008">
    <failure message="expected [true] but found [false]" type="java.lang.AssertionError">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at org.testng.Assert.fail(Assert.java:99)
at org.testng.Assert.failNotEquals(Assert.java:1037)
at org.testng.Assert.assertTrue(Assert.java:45)
at org.testng.Assert.assertTrue(Assert.java:55)
at com.tutorialsninja.testcases.Logintest.verify_Loggingino_the_application_and_browsingback_using_browserback_button(Logintest.java:124)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:568)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:598)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:173)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:824)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:146)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at org.testng.TestRunner.privateRun(TestRunner.java:794)
at org.testng.TestRunner.run(TestRunner.java:596)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:377)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:371)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:332)
at org.testng.SuiteRunner.run(SuiteRunner.java:276)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1212)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1134)
at org.testng.TestNG.runSuites(TestNG.java:1063)
at org.testng.TestNG.run(TestNG.java:1031)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
    </failure>
  </testcase> <!-- verify_Loggingino_the_application_and_browsingback_using_browserback_button -->
  <system-out/>
  <testcase classname="com.tutorialsninja.testcases.Logintest" name="verify_Loggingout_of_the_application_and_browsingback_using_browserback_button" time="9.619"/>
  <system-out/>
  <testcase classname="com.tutorialsninja.testcases.Logintest" name="verify_password_is_not_visible_in_pagesource" time="1.009"/>
  <system-out/>
  <testcase classname="com.tutorialsninja.testcases.Logintest" name="Verify_Logging_into_the_Application_closing_the_Browser_without_loggingout_and_opening_the_application_in_the_Browser_again" time="22.887"/>
  <system-out/>
</testsuite> <!-- com.tutorialsninja.testcases.Logintest -->
